features/step_definitions/init.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 
-When(/I initialize a (.+) application named (.+)/) do |platform, name|
+When(/^I initialize a (.+) application named (.+) with (.+) template$/) do |platform, name, template|
```

Comment:
❤️ You are becoming an acceptance tests expert!

Reply:
It's becoming a joy to write these!